_benchmark/workloads/index.md

```diff
@@ -0,0 +1,249 @@
+          "document-count": 11658903, # Fetch document count from command line
+          "uncompressed-bytes": 1544799789 # Fetch uncompressed bytes from command line
+        }
+      ]
+    }
+  ],
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": 5000
+      },
+      "warmup-time-period": 120,
+      "clients": 8
+    },
+    {
+      "operation": {
+        "name": "query-match-all",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+A workload usually consists of the following elements:
+
+- [indices]({{site.url}}{{site.baseurl}}/benchmark/workloads/indices/): Defines the relevant indices and index templates used for the workload.
+- [corpora]({{site.url}}{{site.baseurl}}/benchmark/workloads/corpora/): Defines all documents corpora used for the workload.
+- `schedule`: Defines in what order the workload performs an action. It can use either the `operations` element to define the operations or place the operations in line.
+- `operations`: **Optional**. Describes which operations are available for this track and how they are parametrized. 
+
+### Indices
+
+To create an index, you need to specify the `name` of the index. If you want to add definitions for your index, use the `body` option and point it to the JSON file containing the index definitions.
+
+### Corpora
+
+The `corpora` element requires the name of the index containing the document corpus, for example, `movies`, and a list of parameters that define the document corpora, which includes:
+
+-  `source-file`: The file name to corresponding documents. When using Benchmark locally, documents are contained in a JSON file. When providing a `base_url`, use a compressed file format; `.zip`, `.bz2`, `.gz,` `.tar`, `.tar.gz`, `.tgz` or `.tar.bz2`. The compressed file must contain one JSON file with the name. 
+-  `document-count`  The number of documents in the `source-file` that determines which client indexes correlate to which part of the document corpus (each N client get on N-th of the document corpus). When using a source that contains a document with a parent-child relationship, specify the number of parent documents. 
+- `uncompressed-bytes` The size in bytes of the source file after decompression, used to show you much disk space the decompressed source file needs. You can also indicate the number of `compressed-bytes`, the size of the source file before decompression, which can help you assess the time it'll take for the workload to ingest the documents
```

Comment:
Are users using the `compressed-bytes` to estimate how long it'll take the cluster to ingest the documents? If so, we should edit to  `... take for the cluster to ingest the documents`

Reply:
That's my understanding. Adjusting...